README.md

```diff
@@ -15,6 +15,7 @@ This repository contains MongoDB Compass application source, all Compass plugins
 - [**@mongodb-js/compass-auto-updates**](packages/compass-auto-updates): Compass Auto Updates Plugin
 - [**@mongodb-js/compass-collection**](packages/compass-collection): Compass Collection Plugin
 - [**@mongodb-js/compass-collection-stats**](packages/compass-collection-stats): Compass Collection Stats Plugin
+- [**@mongodb-js/compass-components**](packages/compass-components): A set of React Components used in Compass
```

Comment:
Probably should be added to the shared libraries and build tools section, not here